samples/data-sealing/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) Open Enclave SDK contributors.
 # Licensed under the MIT License.
 
-cmake_minimum_required(VERSION 3.11)
+cmake_minimum_required(VERSION 3.16)
```

Comment:
For my understanding, what cmake feature being used by this sample is driving the need to require a higher version of cmake? In general, we try not to have different samples requiring different cmake versions to build.